src/datumaro/components/operations.py

```diff
@@ -253,21 +258,26 @@ def get_label(ann):
     }
     label_stat = {
         "count": 0,
-        "distribution": {l.name: [0, 0] for l in labels.items},  # label -> (count, total%)
+        "distribution": defaultdict(lambda: [0, 0]),  # label -> (count, total%)
```

Comment:
If `[0,0]` does not change its length, please use `tuple` or `namedtuple`, Python list is less efficient for handling the fixed length data.

Reply:
It make sense and I agree with you. However, I think we need to give users an appropriate notice before changing this, because someone may already use this output to his/her application.